tests/managed/arrays.cs

```diff
@@ -0,0 +1,27 @@
+#if NON_OBJC_SUPPORTED_TESTS
```

Comment:
maybe `!TARGET_OBJC` ? or something similar.  We'll likely have some cases where other generator or platforms needs customization (temporary or permanent). @rolfbjarne ?